packages/database/src/runPostMigration.js

```diff
@@ -21,14 +21,14 @@ const EXCLUDED_TABLES_FROM_TRIGGER_CREATION = [
 // attempted
 const IMMUTABLE_TABLES = ['ancestor_descendant_relation'];
 
-const SELECT_TABLES_WITHOUT_TRIGGERS = `
+const getSelectForTablesWithoutTriggers = triggerSuffix => `
```

Comment:
Changes in this file are unrelated, just found when I was trying to add both trigger types to ancestor_descendant_relation (ended up reverting that actual change)